main/services.go

```diff
@@ -368,7 +368,7 @@ func getHashFromDataRequest(header http.Header, data []byte) (hash Sha256Sum, er
 			return Sha256Sum{}, err
 		}
 
-		if ENV != PROD_STAGE { // never log original data on prod
+		if !EnvIsProd { // never log original data on prod
 			log.Debugf("sorted compact JSON: %s", string(data))
 		}
```

Comment:
I think if we set the DEBUG log level, we are okay to output data. We already check for c.Debug and can put this to "false" for all production stages.

With that assumption we could remove a lot of conditions: we could set a log level for the logging library on initialization, and just put a Info() or Debug() whereever we feel like it, with no additional checks.
Do you think this would make sense?

Reply:
i would prefer if i could rely on the logging level never to be `debug` on any production stages, if you could assure that

Reply:
Okay, we can take care of that by setting the default debug flag to FALSE, this way it has to be explicitly set.

Reply:
Yes, this was a configuration error, it will be fixed on the next deployment.
![image](https://user-images.githubusercontent.com/33927916/114872143-50540300-9dfa-11eb-96a4-a5627eb7d6bc.png)